draft-ietf-lamps-csr-attestation.md

```diff
@@ -71,115 +70,72 @@ normative:
+This specification defines an ASN.1 structure for 
+remote attestation that can accommodate proprietary and standardized
+attestation mechanisms, as well as an attribute and an extension to carry the structure in PKCS#10 and Certificate
+Request Message Format (CRMF) messages, respectively.
 
 --- middle
 
 # Introduction
 
-When requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to include RATS conceptual messages (see {{Section 8 of RFC9334}}, such as Evidence, Endorsements {{I-D.ietf-rats-endorsements}} and
-Attestation Results {{I-D.ietf-rats-ar4si}}, of the security properties of its environments in which the private keys are stored in that request.
-
-Evidence and Endorsements are appraised by Verifiers, which typically produces Attestation Results that serve as input for validating incoming certificate requests against specified certificate policies.
-Verifiers are associated with Registration Authorities (RAs) or CAs and function as logical entities responsible for processing Evidence and Endorsements in order to produce Attestation Results.
-As remote attestation technology matures, it is natural for a Certification Authority to rely on remote attestation data for proof that the requesting entity is in a state that matches the certificate profile. This is referred to as the RATS Background Check Model, and is illustrated in {{fig-arch-background}}.
-
-Alternatively, the Attester might have a direct connection to a Verifier to which it presents its Evidence and Endorsements, and receives back an Attestation Result signed by the Verifier which it can include directly in the CSR and save the RA / CA from needing a local Verifier. This is referred to as the RATS Passport Model, and is illustrated in {{fig-arch-passport}}.
-
-At the time of writing, the most pressing example of the need for remote attestation in certificate enrollment is the Code-Signing Baseline Requirements (CSBR) document maintained by the CA/Browser Forum {{CSBR}}, which requires compliant CAs to "ensure that a Subscriber’s Private Key is generated, stored,
-and used in a secure environment that has controls to prevent theft or misuse", which is a natural fit to enforce via remote attestation.
+Certification Authorities (CAs) issuing certificates to PKI end entities may require a certificate signing request (CSR) to include verifiable attestations that contain claims regarding the platform used by the end entity to generate the key pair for which a certificate is sought. At the time of writing, the most pressing example of the need for remote attestation in certificate enrollment is the Code-Signing Baseline Requirements (CSBR) document maintained by the CA/Browser Forum [CSBR]. The [CSBR] requires compliant CAs to "ensure that a Subscriber's Private Key is generated, stored, and used in a secure environment that has controls to prevent theft or misuse". This requirement is a natural fit to enforce via remote attestation.
```

Comment:
insert "and claims of attributes of the key pair with respect to its protection, use and extractability" after "... for which a certificate is sought"